src/App.test.jsx

```diff
@@ -1,29 +1,85 @@
 import { render } from '@testing-library/react';
-
 import { useSelector } from 'react-redux';
 
 import App from './App';
 
 jest.mock('react-redux');
 
+function setState(listInfo = {}) {
+  useSelector.mockImplementation((selector) => selector(
+    listInfo,
+  ));
+}
+
 describe('App', () => {
-  useSelector.mockImplementation((selector) => selector({
-    shopName: '',
-    shopType: '',
-    shopAddress: '',
-    restaurants: [
-      {
-        id: 1, name: '마녀주방', type: '한식', address: '서울시 강남구',
-      },
-    ],
-  }));
-
-  it('renders ', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  it('renders title', () => {
+    setState({
+      shopName: '',
+      shopType: '',
+      shopAddress: '',
+      restaurants: [],
+    });
+
     const { getByText } = render((
       <App />
     ));
 
+    expect(getByText(/Restaurants/)).not.toBeNull();
+  });
+
+  it('renders input component', () => {
```

Comment:
복수형으로 적는게 알맞지 않을까요?? 🤔
```suggestion
  it('renders input components', () => {
```